### ISSUE TITLE
iOS: Feeds list section preview artifacts stuck on the screen

### ISSUE THREAD

**Issue by philviso** [brentsimmons/NetNewsWire#1345]
See the screenshots below. I was able to get the section title into a weird state by playing with cutting off the preview right before it activates. It kinda looks like the preview view didn't get cleaned up properly and it was just left on the screen.

Here it is right after I was playing with timings...

![Simulator Screen Shot - iPhone 8 - 2019-11-24 at 07 33 21](https://user-images.githubusercontent.com/347423/69495456-e184ab80-0e8c-11ea-95c8-d9e5616ed5e1.png)

Here it is after collapsing the smart feed section. The empty preview view is just hanging out on the screen.

![Simulator Screen Shot - iPhone 8 - 2019-11-24 at 07 35 00](https://user-images.githubusercontent.com/347423/69495463-f8c39900-0e8c-11ea-8f55-9d57c0a18918.png)

**Comment by vincode-io** [comment]
I've tried to recreate this and haven't had any luck.  Was this a preview for an Account, Folder, or Web Feed? 

 Did you have animations slowed down by any chance?  I'm having a heck of a time interrupting the animation at all.

**Comment by philviso** [comment]
I actually saw this when I was trying to reproduce a different issue. The initial issue I saw was that the separators would show and hide every time I expanded / collapsed the cell. When I was doing that, I was clicking the expand / collapse button sorta fast. I haven't been able to reproduce that, and I also haven't been able to reproduce this either. 

What you see in the first screenshot is the empty preview right on top of the cell that was being previewed (an account cell) - no slow animations on. I was just clicking and holding and either letting go or dragging away right before the preview is displayed.
